packages/deployment-server-ui/src/components/Notification.tsx

```diff
@@ -85,15 +85,18 @@ export const deviceEvent = async (resp: ServerMessage): Promise<void> => {
     }
 
     const deviceTypeName = await typeIdToName(device.deviceTypeId)
+    const title = (<a href={'/devices'} style={{ color: 'black' }}>{`Device ${titleEvent}`}</a>)
     const deviceMessage = (
-      <>
+      <a href={'/devices'} style={{ color: 'black' }}>
```

Comment:
Link

Reply:
Not possible due to the function not being a ReactComponent.

Reply:
Please add a comment then, as we both agree that there are better ways to approach this (i.e. onClick).